### PR TITLE
Clean up terra-mixin dependency

### LINKED ISSUE #1361: Remove cerner-consumer-theme from Base
# Issue Description
This behavior should be supplied by the consumer theme, not by the base component. 
See https://github.com/cerner/terra-core/blob/master/packages/terra-base/src/_themes.scss

## Issue Type

- [ ] New Feature
- [ ] Enhancement
- [ ] Bug
- [x] Other

### PR DESCRIPTION
**Pull request by bjankord**
### Summary
Removed unused mixin package and theme mixins from various modules.
Also resolves #1361

### CODE REVIEW

File: packages/terra-avatar/src/Avatar.scss [hunk 1]
```diff
@@ -1,3 +1,5 @@
+@import '~terra-mixins';
```
Comment (emilyrohrbough) [inline packages/terra-avatar/src/Avatar.scss]: Why did we re-add this?

File: packages/terra-badge/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Removed
```
Comment (JakeLaCombe) [inline packages/terra-badge/CHANGELOG.md]: Is this change version correct?

File: packages/terra-avatar/src/Avatar.scss (hunk 1 above)
Comment (bjankord) [reply packages/terra-avatar/src/Avatar.scss]: Avatar.scss uses a sass function named `inline-svg` that is provided in terra-mixins.

File: packages/terra-badge/CHANGELOG.md (hunk 2 above)
Comment (bjankord) [reply packages/terra-badge/CHANGELOG.md]: I believe so. The terra-mixins package has been removed from terra-badge as it is not used.

**Pull request merged**